lib/sheets-data-builder.rb

```diff
@@ -29,7 +29,7 @@ def url
 
   def grouped_data(refresh = false)
     @grouped_data   = nil if refresh
-    @grouped_data ||= raw_data(refresh).each_slice(GROUP_SIZE)
+    @grouped_data ||= raw_data(refresh).each_slice(GROUP_SIZE - 1)
```

Comment:
I remember we had some potential trouble with this code when we briefly paired on it before the break.

Specifically, the first row of cells would be the full 7 columns long (because it has a column for each of the possible class options), but then all rows afterwards would only be 6 columns long.

I'm not seeing how that is being accounted for in this code. Have you had it successfully work with more than 1 checkin entry in the spreadsheet?